release_images.yml

```diff
@@ -167,41 +167,15 @@ release_images:
       force_release: False
   12:
     framework: "tensorflow"
-    version: "2.7.1"
-    customer_type: "e3"
-    arch_type: "x86"
-    training:
-      device_types: ["cpu", "gpu"]
-      python_versions: ["py38"]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu112"
-      example: False
-      disable_sm_tag: False
-      force_release: False
-  13:
-    framework: "tensorflow"
-    version: "2.7.1"
+    version: "2.8.0"
```

Comment:
Are we releasing only SageMaker DLC?

Reply:
Yes, vulnerabilities are present only in sagemaker DLC.